sdk/messaging/azeventhubs/internal/blob/CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Release History
```

Comment:
Can be removed.  Also be sure to open a tracking issue that we should move to the refactored `azblob` at some point which should make it easy to remove the unnecessary content (it's ~1MB for a handful of APIs).